conf/pipeline/eager/crick.config

```diff
@@ -0,0 +1,52 @@
+ params {
+   config_profile_contact = "Alexandre Gilardet (@alexandregilardet )" '
```

Comment:
Contact name needs to be updated

Reply:
I have put myself down in the config and we will update that field in due course